modules/core/04-channel/keeper/keeper_test.go

```diff
@@ -81,6 +82,73 @@ func (suite *KeeperTestSuite) TestGetAppVersion() {
 	suite.Require().Equal(ibcmock.Version, channelVersion)
 }
 
+// TestGetAllChannelsWithPortPrefix verifies ports are filtered correctly using a port prefix.
+func (suite *KeeperTestSuite) TestGetAllChannelsWithPortPrefix() {
+	tests := []struct {
+		name             string
+		prefix           string
+		allChannels      []types.IdentifiedChannel
+		expectedChannels []types.IdentifiedChannel
+	}{
+		{
+			name:   "transfer channel is retrieved with prefix",
+			prefix: "tra",
+			allChannels: []types.IdentifiedChannel{
+				types.NewIdentifiedChannel(transfertypes.TypeMsgTransfer, ibctesting.FirstChannelID, types.Channel{}),
```

Comment:
haha didn't realize  "TypeMsgTransfer" still exists! It was historically used for to fulfill a "Type()" interface function on sdk.Msg, but since the SDK now uses a gRPC router for routing messages, the type can be inferred from the proto url

I'd use the transfertypes.PortID, we should probably remove unused constants (since "TypeMsgTransfer" has no intended meaning right now)